CHANGELOG-zh-CN.md

```diff
@@ -1,5 +1,65 @@
 # OceanBase Developer Center (ODC) CHANGELOG
 
+## 4.3.1 (2024-07-)
+
+### 功能变化
+
+变更风险管控
+
+- 新增表级别权限管控，允许项目成员对不同表拥有不同操作权限，包括对表的查询、变更和导出操作
+
+会话管理
+
+- 当连接为 OceanBase 4.2.3 和 obproxy 4.2.5 或更高的版本时，使用 Client session 能力进行会话关闭
+- 在 OceanBase 4.2.1 oracle 模式下，使用匿名块进行会话关闭
+
+SQL 诊断
+
+- 支持以图格式查看 OceanBase 的逻辑 SQL 执行计划
```

Comment:
too simple, we do a lot of great job, it is desired to annount more details

Reply:
sure, I have expanded some of the content, please review if it is sufficient